packages/askui-nodejs/src/utils/proxy/proxy-utils.ts

```diff
@@ -0,0 +1,111 @@
+import fs from 'fs';
+import http from 'http';
+import https from 'https';
+import { join } from 'path';
+import dns, { LookupOneOptions } from 'dns';
+
+const proxy = require('proxy');
```

Comment:
Can't you do `import proxy from 'proxy';`?

Reply:
I see. There are no types available. I think we can solve this in a nicer way by simply adding a declaration file.